ethcore/src/blockchain/blockchain.rs

```diff
@@ -586,8 +591,12 @@ impl BlockChain {
 		{
 			let best_block_number = bc.best_block.read().header.number();
 			// Fetch first and best ancient block details
-			let raw_first = bc.db.key_value().get(db::COL_EXTRA, b"first").unwrap().map(|v| v.into_vec());
-			let mut best_ancient = bc.db.key_value().get(db::COL_EXTRA, b"ancient").unwrap().map(|h| H256::from_slice(&h));
+			let raw_first = bc.db.key_value().get(db::COL_EXTRA, b"first")
+				.expect("Low level database error. Some issue with disk?")
```

Comment:
Same as above, we can mention that it's the "first" block and the "best ancient" block below.